tests/Schema/ComparatorTest.php

```diff
@@ -662,22 +662,6 @@ public function testCompareForeignKeyBasedOnPropertiesNotName(): void
         self::assertNull($tableDiff);
     }
 
-    public function testCompareForeignKeyRestrictNoActionAreTheSame(): void
-    {
-        $fk1 = new ForeignKeyConstraint(['foo'], 'bar', ['baz'], 'fk1', ['onDelete' => 'NO ACTION']);
-        $fk2 = new ForeignKeyConstraint(['foo'], 'bar', ['baz'], 'fk1', ['onDelete' => 'RESTRICT']);
-
-        self::assertFalse($this->comparator->diffForeignKey($fk1, $fk2));
-    }
```

Comment:
This test has been reworked into `ForeignKeyConstraintTest:: testCompareRestrictAndNoActionAreTheSame()`.